vars/liskBuild.groovy

```diff
@@ -32,10 +59,10 @@ def call(Map params) {
 	dir(liskBuildDir) {
 		git url: "https://github.com/LiskHQ/lisk-build.git", branch: liskBuildBranch
 		sh """
-		cp "../${liskDir}/release/${liskReleaseName}" "src/"
+		cp "${liskDir}/release/${liskReleaseName}" "src/"
 		bash build.sh -n "${buildNetwork}" -v "${liskVersion}"
 		"""
 	}
 
-	return [version: liskVersion, latestVersion: latestVersion, network: buildNetwork, file: "${liskBuildDir}/release/lisk-${liskVersion}-Linux-x86_64.tar.gz"]
+	return [version: liskVersion, latestVersion: latestVersion, network: buildNetwork, file: "${liskBuildDir}/release/lisk-Linux-x86_64.tar.gz"]
```

Comment:
The output of lisk-build is versioned I think @Nazgolze

Reply:
Ah - the nightly builds

Reply:
@Nazgolze How to resolve this issue? I am in favor to have version there. But with  version `liskInstall.sh` script does not work.

Reply:
@nazarhussain @abelboldu Technically, we don't need to use what gets returned in the jenkins migration job. lisk-build is always going to create "${liskBuildDir}/release/lisk-Linux-x86_64.tar.gz".

So we have it still return "${liskBuildDir}/release/lisk-${liskVersion}-Linux-x86_64.tar.gz" for the nightly builds

Reply:
after talking with @nazarhussain, it's easiest to add a nonVersionedFile field